docs/data/base/components/badge/badge.md

```diff
@@ -1,43 +1,149 @@
 ---
 product: base
-title: Unstyled React Badge component
+title: Unstyled React Badge component and hook
 components: BadgeUnstyled
 githubLabel: 'component: badge'
 packageName: '@mui/base'
 ---
 
 # Unstyled badge
 
-<p class="description">The `BadgeUnstyled` component generates a small label that is attached to its children elements.</p>
+<p class="description">The BadgeUnstyled component generates a small label that is attached to its child element.</p>
 
-```js
+## Introduction
+
+A badge is a small descriptor for UI elements.
+It typically sits on or near an element and indicates the status of that element by displaying a number, icon, or other short set of characters.
+
+The `BadgeUnstyled` component creates a badge that is applied to its child element.
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+## Component
+
+### Anatomy
```

Comment:
I'd say it's more of a "usage" than "anatomy".
IMO the anatomy section is what we have under the "Component slots" heading.

Reply:
That's a good point! I'm going to rearrange these headers accordingly.